osquery/distributed/distributed.cpp

```diff
@@ -183,6 +166,21 @@ Status Distributed::acceptWork(const std::string& work) {
               1,
               "Distributed discovery query does not have complete attributes");
         }
+        if (doesQueryRequireSignature(query)) {
+          if (signatures.count(node.first + "_disc") > 0) {
```

Comment:
This comment is still here, and I think you mentioned wanting to address it, right?